ocaml/xapi/xapi_globs.ml

```diff
@@ -633,6 +633,10 @@ let network_reset_trigger = "/tmp/network-reset"
 
 let first_boot_dir = "/etc/firstboot.d/"
 
+(** {2 Xenopsd metadata persistence} *)
```

Comment:
A `module Xenopsd = struct … end` could save the use of `a_lot_of_underscores` as the module system would provide the name spaces.